### DISCUSSION TITLE
[Bug] 本地的 Ollama 模型无法调用

### DISCUSSION THREAD

**Discussion by jerlinn** [lobehub/lobehub#2284] [General Question] [answered]
### 💻 Operating System

macOS

### 📦 Environment

Docker

### 🌐 Browser

Chrome

### 🐛 Bug Description

Mac 本地 Docker 部署之后，无法调用本地的 ollama 模型。

Connectivity Check 通过：  
![CleanShot 2024-04-28 at 18 24 34@2x](https://github.com/lobehub/lobe-chat/assets/91647085/86b16295-5e09-4520-b87e-0a8c675bdf0c)

本地模型 ：  
![WX20240428-182748@2x](https://github.com/lobehub/lobe-chat/assets/91647085/551b23c2-475f-48ba-a361-d90fb7145eef)

实际使用时，所有 ollama 模型都返回出错提示：  
![WX20240428-182521@2x](https://github.com/lobehub/lobe-chat/assets/91647085/b5502931-1e0f-4826-90a2-e08d742b2b14)

### 📷 Recurrence Steps

正常使用都会必现

### 📝 Additional Information

- ollma 有正常开启，端口也正常
- 无论是否设置环境变量，问题依旧
- 分别尝试了从.zshrc 文件、以及在终端中设置环境变量

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
能否看下控制台的错误？

**Reply by jerlinn** [reply]
![CleanShot 2024-05-05 at 14 23 45@2x](https://github.com/lobehub/lobe-chat/assets/91647085/fa22eaf5-0c48-4cee-8dae-2c7ef712da0f)
控制台的Ollama全跪下，Groq 也只有Mixtral-8x7b 能用

[a comment by an automated account is not shown]

**Comment by rickdgit** [comment]
Got the same error
Console output: 


```javascript
17930-c6532f96900818ef.js:1 Route: [ollama] InvalidOllamaArgs: 
{error: undefined, errorType: 'InvalidOllamaArgs'}
error
: 
undefined
errorType
: 
"InvalidOllamaArgs"
[[Prototype]]
: 
Object
```

**Comment by rickdgit** [comment, marked as answer]
Okey I figure it out
Check the Interface proxy address and make sure it start with 'http://'
